cpp/src/io/statistics/column_stats.h

```diff
@@ -36,8 +36,10 @@ enum statistics_dtype {
 };
 
 struct stats_column_desc {
-  statistics_dtype stats_dtype;    //!< physical data type of column
-  uint32_t num_rows;               //!< number of rows in column
+  statistics_dtype stats_dtype;  //!< physical data type of column
+  uint32_t num_rows;             //!< number of rows in column
+  uint32_t num_values;  //!< Number of data values in column. Different from num_rows in case of
```

Comment:
Where `num_values` is being updated for Parquet ?

Reply:
https://github.com/rapidsai/cudf/blob/ff022e6c1e3fbb75974c0b34a56129b04fffef13/cpp%2Fsrc%2Fio%2Fparquet%2Fpage_enc.cu#L161-L169